engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java

```diff
@@ -535,7 +535,8 @@ public void runLocal(AppFactory appConfig) throws Exception
 
   public URLClassLoader loadDependencies()
   {
-    URLClassLoader cl = URLClassLoader.newInstance(launchDependencies.toArray(new URL[launchDependencies.size()]));
+    ClassLoader currentCl = Thread.currentThread().getContextClassLoader();
+    URLClassLoader cl = URLClassLoader.newInstance(launchDependencies.toArray(new URL[launchDependencies.size()]), currentCl);
```

Comment:
Is this needed if the necessary artifacts are available in the launch dependencies